### ISSUE TITLE
Remove phpjs file

### ISSUE THREAD

**Issue by TimMikeladze** [Tacoma-JS/weatherapp#11]
It may be more convenient for PHP programmers but this is a Javascript project after all and not everyone is coming from a PHP background. The functionality performed in that file can be done through a validation package like check or inline in the code.

**Comment by topcheese** [comment]
Yeah, that kind of threw me for a loop, don't they have separate libraries for that?  I've just grabbed it, but I would tend to agree.  JavaScript has no need for PHP, so it detracts from the purpose of trying to use JavaScript in the first place.  That has nothing to do with solving the original problem. and should be a separate branch.
